corehq/blobs/tests/test_migrate.py

```diff
@@ -158,541 +143,3 @@ def _do_migration(self, doc):
             self.assertEqual(len(exp.external_blobs), num_blobs)
-
-
-class TestSavedExportsMigrations(BaseMigrationTest):
-
-    slug = "saved_exports"
-
-    def test_migrate_happy_path(self):
-        saved = SavedBasicExport(configuration=_mk_config())
-        saved.save()
-        payload = b'binary data not valid utf-8 \xe4\x94'
-        name = saved.get_attachment_name()
-        super(BlobMixin, saved).put_attachment(payload, name)
-        saved.save()
-
-        self.do_migration([saved])
-
-        exp = SavedBasicExport.get(saved._id)
-        self.assertEqual(exp.get_payload(), payload)
-
-    def test_migrate_with_concurrent_modification(self):
-        saved = SavedBasicExport(configuration=_mk_config())
-        saved.save()
-        name = saved.get_attachment_name()
-        new_payload = 'something new'
-        old_payload = 'something old'
-        super(BlobMixin, saved).put_attachment(old_payload, name)
-        super(BlobMixin, saved).put_attachment(old_payload, "other")
-        saved.save()
-        self.assertEqual(len(saved._attachments), 2)
-
-        def modify(doc):
-            doc = SavedBasicExport.get(doc._id)
-            doc.set_payload(new_payload)
-            doc.save()
-
-        self.do_failed_migration({saved: (1, 1)}, modify)
-
-        exp = SavedBasicExport.get(saved._id)
-        self.assertEqual(exp.get_payload(), new_payload)
-        self.assertEqual(exp.fetch_attachment("other"), old_payload)
-
-
-class TestDomainMigrations(BaseMigrationTest):
-
-    slug = "domains"
-
-    def test_migrate_happy_path(self):
-        domain = Domain(name='test_logo_attachment')
-        domain.save()
-        domain_logo = b'logo: binary data not valid utf-8 \xe4\x94'
-        super(BlobMixin, domain).put_attachment(domain_logo, LOGO_ATTACHMENT)
-        domain.save()
-
-        self.do_migration([domain])
-
-        domain = Domain.get(domain._id)
-        self.assertEqual(domain.get_custom_logo()[0], domain_logo)
-        self.assertIsNone(domain._attachments)
-
-    def test_migrate_with_concurrent_modification(self):
-        domain = Domain(name='test_logo_attachment')
-        domain.save()
-        new_logo = b'something new'
-        old_logo = b'something old'
-        super(BlobMixin, domain).put_attachment(old_logo, LOGO_ATTACHMENT)
-        super(BlobMixin, domain).put_attachment(old_logo, 'other')
-        domain.save()
-        self.assertEqual(len(domain._attachments), 2)
-
-        def modify(domain_doc):
-            domain_doc = Domain.get(domain_doc._id)
-            domain_doc.put_attachment(new_logo, LOGO_ATTACHMENT)
-            domain_doc.save()
-
-        self.do_failed_migration({domain: (1, 1)}, modify)
-
-        domain = Domain.get(domain._id)
-        self.assertEqual(domain.get_custom_logo()[0], new_logo)
-        self.assertEqual(domain.fetch_attachment('other'), old_logo)
-
-
-class TestApplicationMigrations(BaseMigrationTest):
```

Comment:
I'd like to keep at least one of these around (commented out) as an example in case we ever need to write another one in the future. You can use a block comment like this:

https://github.com/dimagi/commcare-hq/blob/3a75550444bf94db737794d2b3beb7f7255cdb50/corehq/blobs/tests/test_migrate.py#L159-L165

Comment end delimiter not shown.

Reply:
kept ```TestApplicationMigrations```